roles/seqreports/tasks/1_install.yml

```diff
@@ -0,0 +1,65 @@
+---
+
+- name: set artifact path
+  set_fact:
+    artifact_path: "{{ seqreports_artifacts_path }}/{{ job_id }}"
+
+- name: create directory for artifact
+  file:
+    path: "{{ artifact_path }}"
+    state: directory
+    mode: "0755"
+
+- name: create conda environment for seqreports
+  shell: "{{ anaconda_path }}/bin/conda create -n {{ seqreports_venv_name }} -c anaconda python=3.8 -y"
```

Comment:
I change it. But, at a closer look I got a little bit hesitant on this. In future, when we want to change python version, it will probably be much smoother to take one service at a time. Having one single source of python version suggest to change all at once?

Reply:
sure, that's a good point. Although it may be good practice to try and keep the code reasonably coherent, it shouldn't be an absolute requirement to use the same version everywhere. In that case, I'd suggest overriding the variable in the role needing a different version.